modules/oe_content_entity/tests/src/FunctionalJavascript/DocumentReferenceCorporateEntityUiTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_content_entity\FunctionalJavascript;
+
+/**
+ * Test Document reference content entity UIs.
+ */
+class DocumentReferenceCorporateEntityUiTest extends CorporateEntityUiTestBase {
```

Comment:
Other corporate entity types look like also revisional but they don't have dedicated tests for testing revisions.
Why we have to create a special tests only for Document reference entity type?

Reply:
Actually the same logic as in other corporate entity types is used here https://github.com/openeuropa/oe_content/blob/master/modules/oe_content_entity/tests/src/FunctionalJavascript/ContentEntityUiTest.php#L79.